cli/dxapplet/src/code.sh

```diff
@@ -89,10 +89,22 @@ main() {
             sudo perf record -F $recordFreq -a -g &
         fi
 
+        residuals_flag=""
+        if [ -n $residuals ]; then
```

Comment:
I think `$residuals` will be either `"false"` or `"true"` so `-n` might not be the right test. You can try running the applet with residuals=false and see if it still generates it. LGTM otherwise